package.json

```diff
@@ -19,26 +19,25 @@
   "bugs": "https://github.com/MindscapeHQ/raygun4js/issues",
   "license": "SEE LICENSE IN https://github.com/MindscapeHQ/raygun4js/blob/master/LICENSE",
   "devDependencies": {
-    "@wdio/cli": "^6.1.8",
-    "@wdio/dot-reporter": "^6.1.5",
-    "@wdio/jasmine-framework": "^6.1.8",
-    "@wdio/local-runner": "^6.1.8",
-    "@wdio/selenium-standalone-service": "^6.0.16",
-    "@wdio/spec-reporter": "^6.1.5",
-    "@wdio/static-server-service": "^6.0.16",
-    "@wdio/sync": "^6.1.8",
-    "chromedriver": "^103.0.0",
-    "cross-env": "^7.0.2",
-    "grunt": "^1.1.0",
-    "grunt-contrib-clean": "^2.0.0",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-jshint": "^2.1.0",
-    "grunt-contrib-uglify": "^4.0.1",
+    "@wdio/cli": "^8.11.2",
```

Comment:
What led to all these dependencies needing to be updated?

Reply:
Well!

I needed to update the node version to one which was supported:
1. Period
2. On My machine
3. On Github actions 

I then had to update the testing libraries etc. So that would compile under a supported version of node.

Because of a breaking changes in the wdio & jasmine i needed to update thew tests (`async` and removing .`not.` respectively)